src/openrct2/world/Sprite.h

```diff
@@ -137,6 +137,10 @@ union rct_sprite
     Duck* AsDuck();
     MoneyEffect* AsMoneyEffect();
     Peep* AsPeep();
+    rct_sprite()
+        : pad_00()
+    {
+    }
```

Comment:
Is this necessary to accommodate the fact that `CoordsXYZ` are "~2*" larger than the individual coords previously?

Reply:
It's because cordsxyz can't be trivially constructed. I had another idea how to do it though so might change it.